### PR TITLE
fix(common): table reset always show and pageSizeOptions change

### PR DESCRIPTION
**Pull request by hujiahao-hjh**
## What this PR does / why we need it:
Table reset always show and pageSizeOptions change

## I have checked the following points:
- [x] I18n is finished and updated by cli
- [x] Form fields validation is added and length is limited
- [x] Display normally on small screen
- [x] Display normally when some data is empty or null
- [X] Display normally in english mode

## Does this PR introduce a user interface change?

✅ Yes(screenshot is required)
image
image

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
✅ Yes(version is required)
release/1.5-alpha2

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/common/components/pagination/index.tsx [hunk 1]
```diff
@@ -72,15 +72,13 @@ const Pagination = (pagination: IPaginationProps) => {
...
-      {(pageSizeOptions || [])
-        .filter((item) => item)
-        .map((item: string | number) => {
-          return (
-            <Menu.Item key={item} onClick={() => onChange?.(1, +item)}>
-              <span className="fake-link mr-1">{i18n.t('{size} items / page', { size: item })}</span>
-            </Menu.Item>
-          );
-        })}
+      {PAGINATION.pageSizeOptions.map((item: string | number) => {
```
Comment (McDaddy) [inline shell/app/common/components/pagination/index.tsx:75]: in line `103`， show this dropdown still depend on `pageSizeOptions?.length` .  So I dont understand your change logic

File: shell/app/common/components/table/index.tsx [hunk 2]
```diff
@@ -250,6 +250,10 @@ function WrappedTable<T extends object = any>({
     );
   }, [allColumns, sorterMenu, sort, onRow]);
 
+  const onReload = () => {
+    onChange?.({ current, pageSize }, {}, sort, { action: 'paginate', currentDataSource: [] });
```
Comment (McDaddy) [inline shell/app/common/components/table/index.tsx:254]: why？

File: shell/app/common/components/pagination/index.tsx (hunk 1 above)
Comment (hujiahao-hjh) [reply shell/app/common/components/pagination/index.tsx:65]: I forgot to remove the logic in line 103, and then the pageSizeOptions is fixed.

**Review by McDaddy — APPROVED** [review]

**Pull request merged**